tests/gen_testfiles_scripts/photopair.py

```diff
@@ -35,32 +37,34 @@ def create_tables(dir_name, **kwargs):
     for particle in particle_defs:
         for medium in mediums:
             for param in params:
-                args = {
-                    "parametrization": param,
-                    "particle_def": particle,
-                    "target": medium,
-                    "cuts": None,
-                    "interpolate": False
-                }
-
-                xsection = pp.crosssection.make_crosssection(**args)
-
-                for key in buf:
-                    buf[key][1] = [""]
-
-                    for energy in energies:
-                        if key == "dNdx":
-                            result = [str(xsection.calculate_dNdx(energy))]
-
-                        buf[key][1].append(particle.name)
-                        buf[key][1].append(medium.name)
-                        buf[key][1].append(str(multiplier))
-                        buf[key][1].append(str(energy))
-                        buf[key][1].append(xsection.param_name)
-                        buf[key][1].extend(result)
-                        buf[key][1].append("\n")
-
-                    buf[key][0].write("\t".join(buf[key][1]))
+                for lpm in lpms:
```

Comment:
maybe an additional loop with densities 1 and 2 might be useful to test the LPM calculation?